src/components/App/App.js

```diff
@@ -63,7 +66,7 @@ export default class App extends React.Component {
           grow
           width='100%'
         >
-          <ResultsPanel results={resultsValue} searchedValue={searchValue} loadingResults={loadingResults}/>
+        <ResultsPanel results={resultsValue} searchedValue={searchValue} error={errorHandled} loadingResults={loadingResults}/>
```

Comment:
as you're rendering a single component with many props you might consider grouping its props together.
ex: 
```jsx
<ResultsPanel {...resultsPanelProps} />
```

PS: see comment on `getLocals`